extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetailsHeader.ts

```diff
@@ -190,6 +195,31 @@ export class AssessmentDetailsHeader {
 
 	// function that create target selection dropdown
 	private createTargetTypeContainer(): azdata.FlexContainer {
+		let targetTypes = [constants.SUMMARY_SQLDB_TYPE, constants.SUMMARY_MI_TYPE, constants.SUMMARY_VM_TYPE];
+		if (this._readonly) {
```

Comment:
a little confused about the line 199 to 220. targetTypes is constant? there are only three and appliedForMigrationTargetPlatform only has two: SQLDB and MI. the assessment doesn't apply for SQLVM.

Reply:
When DMA report is imported, it is specific to a target, there we have only one option. When SQLAssessment.json is imported it shows target types more than 1. These two are cases when readonly.

When it is not readonly, we show all 3 target platforms in the dropdown.